posts/exploring-iot-infrastructure-support-in-aws-part1.md

```diff
@@ -0,0 +1,211 @@
+---
+authors:
+- Ben Edridge
+tags:
+- AWS
+- IoT
+- Melbourne
+- Australia
+- Greengrass
+date: 2018-07-19T11:41:57.000Z
+title: "Exploring AWS IoT Core and Greengrass Offerings"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2018/07/aws_iot_esp_device.jpeg
+---
+
+According to [IEEE](https://iot.ieee.org/newsletter/march-2017/three-major-challenges-facing-iot.html) the biggest challenges and issues facing Internet of Things (IoT) are security, privacy, connectivity, compatibility, standardisation and intelligent actions/analysis. These are the issues holding us back from large scale world-wide developments in IoT. A large number of academic research [papers](https://ieeexplore.ieee.org/abstract/document/7823334/) have discussed the security and privacy aspects of IoT deployments. 
+
+The closer and more inter-connected a network is the more strain it puts on developers to perform due diligence and lock down the network infrastructure and maintain ethical privacy practices. AWS provides a number of IoT related services since the release of [AWS IoT Core](https://aws.amazon.com/iot-core/) at the end of 2015. These services are quite complex and try to address a number of these aformentioned challenges. Ideally developers want to spend most of their time on the development of the IoT application and not the supporting infrastructure.
+
+In this two-part series I will look at AWS services provided for IoT and the configuration required. I aim to dissect the structure of AWS IoT Core and Greengrass with a broad focus on full end-to-end usage. I will compare offerings by other cloud providers and note some of the difficulties I’ve had throughout the development process of a proof of concept.
```

Comment:
What is the POC? What proof did you want to do?

Reply:
I hope it's a little clearer now. Let me know what you think.